Source/cmake/OptionsMSVC.cmake

```diff
@@ -192,9 +192,12 @@ string(REPLACE "INCREMENTAL:YES" "INCREMENTAL:NO" replace_CMAKE_EXE_LINKER_FLAGS
 set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO "${replace_CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO} /INCREMENTAL:NO")
 
 if (COMPILER_IS_CLANG_CL)
-    # FIXME: Building with clang-cl seemed to fail with 128 bit int support
-    set(HAVE_INT128_T OFF)
-    list(REMOVE_ITEM _WEBKIT_CONFIG_FILE_VARIABLES HAVE_INT128_T)
+    get_filename_component(CMAKE_CXX_COMPILER_DIR ${CMAKE_CXX_COMPILER} DIRECTORY)
+    string(REGEX MATCH "^[0-9]+" CMAKE_CXX_COMPILER_MAJOR_VERSION ${CMAKE_CXX_COMPILER_VERSION})
+    string(JOIN "/" CLANG_BUILTINS_LIBRARY
+        ${CMAKE_CXX_COMPILER_DIR} "../lib/clang" ${CMAKE_CXX_COMPILER_MAJOR_VERSION} "lib/windows/clang_rt.builtins-x86_64.lib")
+    string(APPEND CMAKE_SHARED_LINKER_FLAGS " \"${CLANG_BUILTINS_LIBRARY}\"")
+    string(APPEND CMAKE_EXE_LINKER_FLAGS " \"${CLANG_BUILTINS_LIBRARY}\"")
```

Comment:
Can we use `add_library` here?
https://cmake.org/cmake/help/latest/command/add_library.html

Reply:
I'm not sure how that would work; I'm just doing the same thing we do for ASan:
https://github.com/WebKit/WebKit/blob/main/Source/cmake/WebKitCompilerFlags.cmake#L264